src/chatnificent/layout.py

```diff
@@ -516,7 +518,7 @@ def get_external_stylesheets(self) -> List[Union[str, Dict]]:
     def build_layout(self) -> DashComponent:
         """Complete Mantine layout - wraps MantineProvider around Bootstrap structure."""
         return self.dmc.MantineProvider(
-            theme={"colorScheme": self.theme_name or "light"},
+            forceColorScheme= "light",
```

Comment:
`forceColorScheme=self.theme_name` should work. I just tried it.

`app = Chatnificent(layout=chat.layout.Mantine("dark")` for example